backend/controllers/Event.js

```diff
@@ -23,42 +22,50 @@ const createEvent = async (req, res) => {
     const creatorParticipation = new EventParticipation({
       eventId: event._id,
       userId: creatorId,
-      status: "accepted",
+      status: 'accepted',
     });
     await creatorParticipation.save();
 
-    //Invited is an array of emails
+    // Invited is an array of emails
     if (invited) {
       // Create a new event participation for the invited users by mapping
-
       const invitedParticipation = invited.map(
-        async (email) => await invite(email, event._id) // a bunch of promises in an array
+        async email => await invite(email, event._id), // a bunch of promises in an array
       );
-
-      //make sure all promise is fullfilled
+      //Make sure all promise is fullfilled
       await Promise.all(invitedParticipation);
     }
-
     res.status(201).json({ event: event });
   } catch (error) {
     console.log(`Failed to create event: ${error}`);
     res.status(500).json({ error: error });
   }
 };
 
-// get all events function Phuoc
-// Lay array of events ( each event is a time period). then show that period on calendar
-const getAllEvents = async (req, res) => {};
+/**
+ * Controller function to get events of a user
+ * Name: getAllEvents
+ * Asignee: Phuoc
+ * req: request object containing user id.
+ * res: response object to return status 200 and events array.
+ */
```

Comment:
Thanks for adding these comments em. Don't forget to add them to your existing controllers, not just the one assigned to your teammates